sdk-extensions/logging/src/main/java/io/opentelemetry/sdk/logging/data/LogRecordBuilder.java

```diff
@@ -7,21 +7,36 @@
 
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.resources.Resource;
 import java.util.concurrent.TimeUnit;
 
 public final class LogRecordBuilder {
+  private Resource resource;
+  private InstrumentationLibraryInfo instrumentationLibraryInfo;
   private long timeUnixNano;
   private String traceId = "";
   private String spanId = "";
   private int flags;
   private LogRecord.Severity severity = LogRecord.Severity.UNDEFINED_SEVERITY_NUMBER;
   private String severityText;
   private String name;
-  private AnyValue body = AnyValue.stringAnyValue("");
+  private String body = "";
```

Comment:
interesting default. Does the proto require the body to have a value, or can it be proto-null?

Reply:
Body is optional: https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L116


However, I'm very curious why we dropped `AnyValue` for `String` here.   I'd prefer we keep body as a structured value (with convenience methods to use a string-any value) vs. removing the flexibility.

Reply:
I want to only add the complexity/flexibility if we have an actual use-case. Let's start with String only, please.

Reply:
At a bare minimum, I'd ask that you attend the logging SiG with this feedback on the beta model and suggestions around SDK integrations.

However, regarding use cases, what's the plan for logging exceptions in Java in OTLP?  Naively, I thought body would be an excellent opportunity to define something better than a big long string.

Reply:
I believe exceptions go into the attributes not the body in Java logging.